apps/www/_blog/2022-08-09-slack-consolidate-slackbot-to-consolidate-messages.mdx

```diff
@@ -405,7 +405,7 @@ if __name__ == '__main__':
 
 Of course, we aren't doing proper Python example if we don't make a test notebook available:
 
-[![Open in Colab](/images/blog/2022-august/colab-badge.svg)](https://colab.research.google.com/github/mansueli/SlackConsolidate/blob/main/SlackConsolidate_bot.ipynb)
+[![Open in Colab](/images/blog/slackbot-consilidate/colab-badge.svg)](https://colab.research.google.com/github/mansueli/SlackConsolidate/blob/main/SlackConsolidate_bot.ipynb)
```

Comment:
@mansueli just making sure you meant to write `consilidate`.

Reply:
The path was wrong and everything within the blog was consistent with the typo. It should be all fixed now.

Reply:
@mansueli nice one!